### PR TITLE
chore: Update JSAPI

### PR DESCRIPTION
**Pull request by kellyhutchins**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -101,5 +101,8 @@
   "packageManager": "npm@8.19.4",
   "volta": {
     "node": "16.20.0"
+  },
+  "dependencies": {
+    "vitest": "^2.0.5"
```
Comment (rslibed) [inline package.json:106]: was this change intended?

**Review by rslibed — APPROVED** [review]

**Comment by kellyhutchins** [comment]
@rslibed  accidentally clicked "resolve conversation" link in regards to vitest. In my testing the app wouldn't build without that. I should probably make it a dev dependency. I can remove if it builds for everyone else w/o that

**Comment by rslibed** [comment]
> @rslibed accidentally clicked "resolve conversation" link in regards to vitest. In my testing the app wouldn't build without that. I should probably make it a dev dependency. I can remove if it builds for everyone else w/o that

Hmmm I haven't ran into that issue, but looks like the tests are failing at the installation stage due to out of sync package.json/package-lock.json files

**Comment by kellyhutchins** [comment]
closing this PR i'll make a new one
